docs/en/writing-running-appium/tutorial/swipe/ios-mobile-element-search.md

```diff
@@ -0,0 +1,80 @@
+## iOS 'mobile:': Element search swipe
+
+To search the element on screen or scrollView use:
+- element id with 'name' argument
+- or specify NSPredicate string using 'predicateString' argument
+
+NSPredicate examples available [here](../../ios/ios-predicate.md)
+
+NSPredicate covers element id search with 'name ==
+accessibilityIdentifier' predicate string.
+
+```java
+String pre = "label == 'exact_text'";
+MobileElement el = (MobileElement) driver.findElement(MobileBy.id("element_id"));
+
+mobileScrollToElementIOS(el, pre);
+
+/**
+ * Performs element scroll by predicate string
+ *
+ * @param el  the element to scroll
+ * @param pre the predicate string
+ * @version java-client: 7.3.0
+ **/
+public void mobileScrollToElementIOS(MobileElement el, String pre) {
+    System.out.println("mobileScrollToElementIOS(): pre: '" + pre + "'"); // always log your actions
+
+    // Animation default time:
+    //  - iOS: 200 ms
+    // final value depends on your app and could be greater
+    final int ANIMATION_TIME = 200; // ms
+    final HashMap<String, String> scrollObject = new HashMap<String, String>();
+    scrollObject.put("element", el.getId());
+    scrollObject.put("predicateString", pre);
+    try {
+        driver.executeScript("mobile:scroll", scrollObject);
+        Thread.sleep(ANIMATION_TIME); // always allow swipe action to complete
+    } catch (Exception e) {
+        System.err.println("mobileScrollToElementIOS(): FAILED\n" + e.getMessage());
+        return;
+    }
+}
+```
+
+'mobileScrollToElementIOS' works UNRELIABLE and often missing needed
+element while scrolling. Specially in complex views. Sometimes helps set
+'simpleIsVisibleCheck' capability to true.
```

Comment:
Sometimes setting `simpleIsVisibleCheck` capability to `true` helps to workaround such issue